src/imagemagick.cc

```diff
@@ -546,6 +566,10 @@ NAN_METHOD(Convert) {
     context->filter = !filterValue->IsUndefined() ?
         *NanAsciiString(filterValue) : "";
 
+    Local<Value> backgroundValue = obj->Get( NanNew<String>("background") );
```

Comment:
Change `NanNew<String>("background")` to `Nan::New<String>("background").ToLocalChecked()`